src/main/daml/Daml/Finance/Interface/Holding/Account.daml

```diff
@@ -50,7 +51,9 @@ data View = View
       -- ^ Human readable description of the account.
     holdingFactoryCid : ContractId Holding.F
       -- ^ Associated holding factory.
-  deriving (Eq, Ord, Show)
+    transferors : Transferors
```

Comment:
I wonder if we should just call this `controllers` or `authorizers`?

Reply:
I changed to controllers, with the new data type it is descriptive enough.